dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.messaging-3.0.feature

```diff
@@ -2,6 +2,7 @@
 symbolicName=io.openliberty.jakarta.messaging-3.0
 singleton=true
 -bundles=io.openliberty.jakarta.messaging.3.0; location:="dev/api/spec/,lib/"; mavenCoordinates="jakarta.jms:jakarta.jms-api:3.0.0"
+-features=com.ibm.websphere.appserver.eeCompatible-9.0
```

Comment:
Should we make a jms-2.0 API feature with eeCompatible in it and have features depend on it instead of pulling in the bundle directly?

Reply:
> Should we make a jms-2.0 API feature with eeCompatible in it and have features depend on it instead of pulling in the bundle directly?

Probably.  Will add to new issue to investigate.